content/integrations/aws.md

```diff
@@ -182,21 +154,21 @@ written a bit about this [here][7],  especially in relation to CloudWatch.
 
 
 
-## Missing metrics?
+**Missing metrics?**
 
 CloudWatch's api returns only metrics with datapoints, so if for instance an ELB has no attached instances, it is expected not to see metrics related to this ELB in Datadog.
 
 
 
-## Wrong count of aws.elb.healthy_host_count?
+**Wrong count of aws.elb.healthy_host_count?**
 
 When the Cross-Zone Load Balancing option is enabled on an ELB, all the instances attached to this ELB are considered part of all A-Zs (on cloudwatch’s side), so if you have 2 instances in 1a and 3 in ab, the metric will display 5 instances per A-Z.
 As this can be counter-intuitive, we’ve added a new metric, aws.elb.host_count, that displays the count of healthy instances per AZ, regardless of if this Cross-Zone Load Balancing option is enabled or not.
 This metric should have value you would expect.
 
 
 
-## Duplicated hosts when installing the agent?
+**Duplicated hosts when installing the agent?**
```

Comment:
@technovangelist why do we want to update that? It's very convenient to be able to reference to those sections in one click :)